src/main/java/ladder/util/StringUtils.java

```diff
@@ -0,0 +1,10 @@
+package ladder.util;
+
+public class StringUtils {
+    public StringUtils() {
+    }
+
+    public static boolean validEmpty(String target) {
+        return target == null || target == "" || "".equals(target.trim());
```

Comment:
```suggestion
        rreturn Objects.isNull(target) || target.equals(EMPTY_STRING) || EMPTY_STRING.equals(target.trim());
```
String 비교를 == 연산자로 비교한다면 정확한 결과를 얻지 못할거에요. 
혹시 모르고 계셨다면 이유를 찾아보세요 :)